main/plugins/org.talend.sdi.designer.components/components/sOGRInfoInput/sOGRInfoInput_begin.javajet

```diff
@@ -54,7 +54,7 @@
     String fidColumn_<%=cid %> = "";
     String geomColumn_<%=cid %> = "";
     String columnsDef_<%=cid %> = "";
-    long nbFeatures_<%=cid %> = 0;
+    int nbFeatures_<%=cid %> = 0;
```

Comment:
Sounds bizarre regarding https://gdal.org/java/org/gdal/ogr/Layer.html#GetFeatureCount-- ? or this depends on OGC version ?

Reply:
I am going to test this fix :)

Reply:
you are right, this is not the correct fix! I will try another fix.

Reply:
Maybe the schema of row8 ?

Reply:
I think the sOGRInfoInput was not uptodate in my metadata_crwaler job. I updated it (new component showed the correct type in the schema). In my job, I can't update the sOGRInfoInput schema.